src/test/scala/mimir/demo/CureScenario.scala

```diff
@@ -48,14 +48,25 @@ object CureScenario
         }
         time(s"Materialize '$basename'"){
           //XXX: there is a problem with materialized views and MV lens
-          //       that needs to be fixed - something with the col det bit vector
-          //      it looks like MIMIR_COL_DET_MIMIR_ROWID_0 is in the non-materialized VIEW's TAINT metadata
-          //      but not in the materialized version of the same view
+          //      that needs to be addressed more thoroughly - 
+          //      something with col det bit vector and taint cols
+          //      it looks like taint cols for prov (MIMIR_COL_DET_MIMIR_ROWID_X) 
+          //      are in the non-materialized VIEW's TAINT metadata
+          //      but not in the materialized version of the same view.
+          //      I added a hack in CTPercolator (see line 530)
+          //      but is likely the wrong solution
```

Comment:
Agreed.  The fix should be in the ViewManager... if it's not materializing those columns, they should be getting materialized.